website/docs/docs/ai/index.md

```diff
@@ -51,19 +60,56 @@ export PACT_BROKER_TOKEN="YOUR_TOKEN"
 
 #### Local configuration file
 
-You can also configure the CLI using the following commands:
+Configure the CLI using the following commands:
 
 ```
-pactflow-ai config set url https://YOUR_ACCOUNT.pactflow.io
-pactflow-ai config set token YOUR_TOKEN
+pactflow-ai config set broker.base-url https://YOUR_ACCOUNT.pactflow.io
+pactflow-ai config set broker.auth.token YOUR_TOKEN
 ```
 
-This will create a `.pactflow.toml` file, which should be added to your `.gitignore`.
+This creates a `.pact.toml` file, which should be added to your `.gitignore`.
 
 #### User configuration file
 
-Alternatively, the user configuration file can be stored in a configuration directory for PactFlow. The paths are `~/.config/pactflow/config.toml` on Unix, and `%APPDATA%\pactflow\config.toml` on Windows.
+Alternatively, store the user configuration file in `~/.config/pact/config.toml` (Unix), and `%APPDATA%\pact\config.toml` (Windows).
+
+Pass the `--user` flag when running `pactflow-ai config set` to write to this file automatically.
+
+## Usage Limits
+
+Access to the feature is metered based on the number of tests generated in a 30-day rolling window. Credits are allocated as follows:
+
+1. Starter Plans: 10 credits per month
+2. All other plans: 10 credits per purchased user per month
+
+Credits are allocated to the account, not individual users.
```

Comment:
Is there a better word than "account" here? Just to avoid confusing between a user's account vs a tenant.